youtube_dl/extractor/francetv.py

```diff
@@ -90,17 +91,47 @@ def _extract_video(self, video_id, catalogue=None):
         # Videos are identified by idDiffusion so catalogue part is optional.
         # However when provided, some extra formats may be returned so we pass
         # it if available.
-        info = self._download_json(
-            'https://sivideo.webservices.francetelevisions.fr/tools/getInfosOeuvre/v2/',
-            video_id, 'Downloading video JSON', query={
-                'idDiffusion': video_id,
-                'catalogue': catalogue or '',
-            })
-
-        if info.get('status') == 'NOK':
+
+        info = {
+            'title': None,
+            'subtitle': None,
+            'image': None,
+            'subtitles': {},
+            'duration': None,
+            'videos': [],
+            'formats': [],
+        }
```

Comment:
It seems rather silly to stuff extracted information into an `info` dictionary only to extract each key back one by one at the end. The churn of changing `formats` into `info['formats']` later in the extractor inflates the diff size and makes code harder to review without actually accomplishing anything.